### PR TITLE
Periodically Check for a new Fedora version and create a new CDI

### PR DESCRIPTION
**Pull request by shwetaap**
Periodically Check for a new Fedora version and create a new Containerdiskimage for it in the registry

Signed-off-by: Shweta Padubidri <spadubid@redhat.com>

### CODE REVIEW

**Comment by rmohr** [comment]
/cc @fgimenez 

If there are no annotations on the job, shoulld the testgrid-config pick this job up?

File: github/ci/prow/files/jobs/kubevirt/common-templates/common-templates-periodics.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+periodics:
+- name: periodic-common-templates-nightly-image-builder
+  cron: "*/30 * * * *"
+  decorate: true
```
Comment (fgimenez) [inline github/ci/prow/files/jobs/kubevirt/common-templates/common-templates-periodics.yaml:4]:
```suggestion
  annotations:
    testgrid-create-test-group: "false" 
  decorate: true
```

**Review by fgimenez — CHANGES_REQUESTED** [review]

**Comment by fgimenez** [comment]
> If there are no annotations on the job, shoulld the testgrid-config pick this job up?

Good catch, periodics should either define the dashboard to which the job should be added or declare that we don't want it to be shown.  I've proposed a small change to not include this job in testgrid, it should fix https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/pr-logs/pull/kubevirt_project-infra/871/pull-project-infra-check-testgrid-config/1352042776916660225 too.

**Review by fgimenez — APPROVED** [review]
/lgtm

**Comment by rmohr** [comment]
/approve

the job is not ready. Appriving to allow @shwetaap iterating on it.

**Pull request merged**